PubNub/Data/Service Objects/PNSubscribeStatus.h

```diff
@@ -72,6 +74,9 @@
  */
 @property (nonatomic, readonly, strong) NSNumber *lastTimeToken;
 
+@property (nonatomic, readonly, strong) NSNumber *lastRegion;
+@property (nonatomic, readonly, strong) NSNumber *currentRegion;
```

Comment:
@jzucker2 Same as note above about whether it is advisability to expose this information to the user (and suggestion how we can do it if it will be required).